test/integration/amphtml-ssg/pages/blog/[slug].js

```diff
@@ -4,9 +4,10 @@ export const config = {
   amp: 'hybrid',
 }
 
-export const getStaticProps = () => {
+export const getStaticProps = (ctx) => {
   return {
     props: {
+      slug: (ctx && ctx.params && ctx.params.slug) || null,
```

Comment:
```suggestion
      slug: ctx.params?.slug || null,
```

Reply:
Oh, I didn't notice you use optional chaining, nice!